test/module/irohad/multi_sig_transactions/mst_test_helpers.hpp

```diff
@@ -89,10 +91,11 @@ auto addSignaturesFromKeyPairs(Batch &&batch,
   return std::forward<Batch>(batch);
 }
 
-inline auto makeSignature(const std::string &sign,
-                          const std::string &public_key) {
-  return std::make_pair(shared_model::crypto::Signed(sign),
-                        shared_model::crypto::PublicKey(public_key));
+inline auto makeSignature(
+    shared_model::interface::types::SignedHexStringView sign,
+    shared_model::interface::types::PublicKeyHexStringView public_key) {
+  return std::make_pair(std::string_view{sign},
```

Comment:
Is it safe to store the view for signature, and make a string copy for public key? If not, then maybe make an explicit copy of signature as well.

Reply:
I agree, in tests we'd prefer safety over speed.